docs(README.md): update contributing and contributors

manudeli opened the pull request:
# Overview

<!--
    A clear and concise description of what this pr is about.
 -->

I added contributing section and contributors
Thank you @okinawaa's adding CONTRIBUTING.md

## PR Checklist

- [x] I have written documents and tests, if needed.

minsoo-web reviewed (approved):
How about add contributing guide link in PR template 🤔? 

reference is toss/slash!!

manudeli commented:
I updated Pull Request template in [05818d3](https://github.com/suspensive/react/pull/225/commits/05818d3e828056e3367ffd42b3698adc144539c1). and I added you as co-author
